packages/sdk-api/src/index.ts

```diff
@@ -3,75 +3,75 @@
  * DO NOT MODIFY - This file has been generated using @tzkt/oazapfts.
  * See https://www.npmjs.com/package/@tzkt/oazapfts
  */
-import * as Oazapfts from '@tzkt/oazapfts/lib/runtime';
-import * as QS from '@tzkt/oazapfts/lib/runtime/query';
-
-import QueryParamsParsers from './queryParamParsers';
```

Comment:
how's it going to make requests without parsing - i.e. making strings out of objects?